### PR TITLE
Dutch, Norwegian, Russian and Welsh

### PR DESCRIPTION
**Pull request by philclifford**
A little knowledge is a dangerous thing, but a little ML and
crowdsourcing can make you feel less nervous.

### CODE REVIEW

**Comment by philclifford** [comment]
Would definitely prefer these not to be merged without native speakers reviewing them.

File: assets/i18n/ru.po [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+"Plural-Forms: nplurals=4; plural=((n%10==1 && n%100!=11) ? 0 : ((n%10 >= 2 && n%10 <=4 && (n%100 < 12 || n%100 > 14)) ? 1 : ((n%10 == 0 || (n%10 >= 5 && n%10 <=9)) || (n%100 >= 11 && n%100 <= 14)) ? 2 : 3));\n"
+"Language: ru\n"
+"X-Crowdin-Project: quickgui\n"
+"X-Crowdin-Project-ID: 485591\n"
+"X-Crowdin-Language: ru\n"
+"X-Crowdin-File: quickgui.pot\n"
+"X-Crowdin-File-ID: 1\n"
+
+msgid "Quickgui : a Flutter frontend for Quickget and Quickemu"
+msgstr "Quickgui : Флэттер фронтенд для Quickget и Quickemu"
```
Comment (Sadykhzadeh) [inline assets/i18n/ru.po:22]: It would be better to write "Flutter фронтент Quickget и Quickemu"

File: assets/i18n/ru.po [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+msgstr "ОК"
+
+msgid "Operating system"
+msgstr "Операционная система"
+
+msgid "Search operating system"
+msgstr "Поиск операционной системы"
+
+msgid "Search option"
+msgstr "Опция поиска"
```
Comment (Sadykhzadeh) [inline assets/i18n/ru.po:91]: From my perspective, "Параметры поиска" would be much better.

File: assets/i18n/ru.po [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+msgstr "Target folder : {0}"
+
+msgid "Use SPICE display"
+msgstr "Использовать экран SPICE"
+
+msgid "Using SPICE display"
+msgstr "Использование SPICE дисплея"
+
+msgid "Version"
+msgstr "Версии"
```
Comment (Sadykhzadeh) [inline assets/i18n/ru.po:124]: "Версии" is in plural, I think translating it into "Версия" would be more appropriate option.

**Review by Sadykhzadeh — CHANGES_REQUESTED** [review]
Overall, the Russian translation is awesome, just some fixes that I commented are recommended to be fixed ^_^

File: assets/i18n/ru.po (hunk 2 above)
Comment (philclifford) [reply assets/i18n/ru.po:91]: Thanks.  In the context this string is used as "Search for the option" mainly (but not only) for windows language - hence "option".  Of course I'll take your perspective with many thanks.

**Comment by philclifford** [comment]
@Sadykhzadeh Thank you so much for your kind words, and for taking the time to improve mine.

**Pull request merged**